ipympl/_version.py

```diff
@@ -1,3 +1,10 @@
-version_info = (0, 8, 8)
+version_info = (0, 8, 9)
 __version__ = '.'.join(map(str, version_info))
-js_semver = '^0.10.5'
+
+
+# The versions of protocol of communication with the frontend that this python verison knows
+# how to speak. See counterpart in the src/version.ts file.
+# These should not be changed unless we introduce changes to communication between
+# frontend and backend.
+__TOOLBAR_MODEL_VERSION__ = "^0.10.5"
+__CANVAS_MODEL_VERSION__ = "^0.10.5"
```

Comment:
It might be easier to track if we have one same version for all models?

Reply:
Yeah that's probably a good call. One isn't really functional without the other so no need to have them separate.